guide/creating_element/testing.md

```diff
@@ -144,6 +144,54 @@ end
 
 Once again, we are comparing the pipeline's output with our reference file to assert that everything works correctly.
 
+### Customizing pipeline module
+
+[`Membrane.Testing.Pipeline`](https://hexdocs.pm/membrane_core/Membrane.Testing.Pipeline.html) allows us to customize every callback of [`Membrane.Pipeline`](https://hexdocs.pm/membrane_core/Membrane.Pipeline.html). Let's create Pipeline that will answer to ping messages. To do this, we must define our custom module.
+
+```elixir
+defmodule MockPipeline do
+  use Membrane.Pipeline
+  
+  def handle_other({:ping, from}, state) do
+      send(from, :pong)
+      {:ok, state}
+  end
+  
+  def handle_other(_message, state),
+    do: {:ok, state}
+end
+```
+
+In order to use callbacks defined in `MockPipeline`, we have to specify it in `Options`.
+
+```elixir
+Pipeline.start_link(%Pipeline.Options{
+  elements: [
+    file_src: %Membrane.Element.File.Source{location: in_path},
+    decoder: Membrane.Element.AAC.Decoder,
+    sink: %Membrane.Element.File.Sink{location: out_path}
+  ],
+  module: MockPipeline
+})
+```
+
+Let's create another test case to try out new functionality.
+
+```elixir
+describe "Decoding Pipeline should" do
+  test "Decode AAC file" do
+    ...
+  end
+  
+  test "Answer with :pong message" do
+    assert {:ok, pid} =  Pipeline.start_link(%Pipeline.Options{...})
+    assert Pipeline.play(pid) == :ok # Start the pipeline
+    send(pid, {:ping, self()}) 
```

Comment:
Maybe instead of `:ping` and `:ping` we could pretend that we are sending a message that pipeline will respond to with some data present in the pipeline internal state?